index.Rmd

```diff
@@ -27,10 +30,92 @@ _Programming skills_
 
 ## Learning Objectives {-}
 
-<!-- Learning objectives for this activity come from the [Genetics Core Competencies](https://genetics-gsa.org/education/genetics-learning-framework/): -->
+- Open RStudio and locate important panels (R console, Help)
+- Run commands in the R console
+- Explain what vectors and dataframes are in R
+- Use common commands to manipulate R objects (e.g. assigning, printing, subsetting, mathematical operations)
+- Carry out a basic exploration of rectangular data using base R commands (e.g. `dim`, `summary`, `str`)
```

Comment:
How about structuring to a short phrase with details in parentheses? e.g.
- Open RStudio on AnVIL (e.g. set up account, locate R console and Help panels)
- Practice working in R Console (e.g. run commands)
- Use swirl to learn basic R commands (e.g. assign, print, subset, math operations)
- Explore a dataset using base R commands (e.g. `dim`, `summary`, `str`)